src/include/duckdb/function/table/arrow.hpp

```diff
@@ -18,11 +18,20 @@ namespace duckdb {
 // Arrow Variable Size Types
 //===--------------------------------------------------------------------===//
 enum class ArrowVariableSizeType : uint8_t { FIXED_SIZE = 0, NORMAL = 1, SUPER_SIZE = 2 };
+
+//===--------------------------------------------------------------------===//
+// Arrow Time/Date Types
+//===--------------------------------------------------------------------===//
+enum class ArrowDateTimeType : uint8_t { MLS = 0, MCS = 1, NS = 2, SEC = 3, DAYS = 4 };
```

Comment:
Maybe write out the type entirely (`ArrowDateTimeType::MICROSECONDS`, `ArrowDateTimeType::MILLISECONDS`). No need to save on characters.